build.xml

```diff
@@ -14,7 +14,7 @@
   <import file="build-res/subfloor-pkg.xml"/>
 
   <!-- Every non-api shim module. These will be compiled/resolved after api. -->
-  <property name="shim-non-api-modules" value="hive-jdbc,hadoop-20,cdh47,cdh52,hdp21,mapr31,mapr401" />
+  <property name="shim-non-api-modules" value="hive-jdbc,hadoop-20,cdh47,cdh52,hdp21,hdp22,mapr31,mapr401" />
```

Comment:
I'm not sure about this. AFAIK, we just adding hdp22 shim for now.
@mbradbury2014 Should we just add the hdp22 shim or replace hdp21?